modules/bitgo/test/v2/unit/lightning.ts

```diff
@@ -84,4 +84,33 @@ describe('lightning API requests', function () {
     assert.deepStrictEqual(res, fixtures.invoices);
     scope.done();
   });
+  
+  it('should decode lnurl-pay', async function () {
+    const lnurl = 'LNURL1DP68GURN8GHJ7UM9WFMXJCM99E3K7MF0V9CXJ0M385EKVCENXC6R2C35XVUKXEFCV5MKVV34X5EKZD3EV56NYD3HXQURZEPEXEJXXEPNXSCRVWFNV9NXZCN9XQ6XYEFHVGCXXCMYXYMNSERXFQ5FNS';
+    const url = 'https://service.com';
+    const balanceScope = nock(bgUrl).get(`/api/v2/wallet/${wallet.id()}/lightning/balance`).reply(200, {
+      balance: 6000,
+      availableBalance: 5679,
+      maximumBalance: 1e8,
+    });
+    const scope = nock(url).get('/api').query({ q: '3fc3645b439ce8e7f2553a69e5267081d96dcd340693afabe04be7b0ccd178df' }).reply(200, fixtures.decodedLnurl);
+    const res = await wallet.lightning().decodeLnurlPay(lnurl);
+
+    assert.deepStrictEqual(res, { ...fixtures.decodedLnurl, maxSendable: 5679000 });
+    balanceScope.done();
+    scope.done();
+  });
+
+  it('should process lnurl invoice payment', async function () {
+    const callback = 'https://service.com/api?q=3fc3645b439ce8e7f2553a69e5267081d96dcd340693afabe04be7b0ccd178df';
+    const invoice = 'lntb20m1pvjluezsp5zyg3zyg3zyg3zyg3zyg3zyg3zyg3zyg3zyg3zyg3zyg3zyg3zygshp58yjmdan79s6qqdhdzgynm4zwqd5d7xmw5fk98klysy043l2ahrqspp5qqqsyqcyq5rqwzqfqqqsyqcyq5rqwzqfqqqsyqcyq5rqwzqfqypqfpp3x9et2e20v6pu37c5d9vax37wxq72un989qrsgqdj545axuxtnfemtpwkc45hx9d2ft7x04mt8q7y6t0k2dge9e7h8kpy9p34ytyslj3yu569aalz2xdk8xkd7ltxqld94u8h2esmsmacgpghe9k8';
```

Comment:
Similar situation here - the opaque blob here makes it difficult to modify this test. This should be a generated value instead.

Reply:
the invoice is from https://github.com/lightning/bolts/blob/master/11-payment-encoding.md bolt11 spec. Do we still want to generate an invoice for ourselves?

Reply:
please put it in a JSON file 

```
{
  "comment": "taken from https://...",
  "data": "lnbt20..."
}
```